setup.py

```diff
@@ -1,4 +1,6 @@
 from setuptools import setup, find_packages
+from Cython.Build import cythonize
+import numpy as np
```

Comment:
These two imports happen before the `install_requires` is even looked at. So installation will if fail Cython or numpy is not installed. The numpy import can be deferred by making `include_dirs` an iterable, e.g. like this, I think:

```python
def include_dirs():
    import numpy as np
    yield np.get_include()

setup(
   # ....
   include_dirs=include_dirs(),
   # ...
)
```

But I have never found out how to defer the Cython import until after Cython was instsalled... I guess this is why the Cython docu recommends to ship the .c sources

Reply:
Of course this is no problem if a person does `pip install -r requirements.txt` before, or she is working on anaconda .. but as soon as `pyeventio` is a dependency of another project which is testing on travis using a vanilla python (not anaconda)  ... then simple installation with `pip git+https:github.com/.../pyeventio` will most probably fail.